packages/client/src/v2-events/layouts/EventOverview/index.tsx

```diff
@@ -11,77 +11,207 @@
 
 import React from 'react'
 import { defineMessages, useIntl } from 'react-intl'
-import { useNavigate } from 'react-router-dom'
-import { useTypedParams } from 'react-router-typesafe-routes/dom'
-import { applyDraftToEventIndex, deepDropNulls } from '@opencrvs/commons/client'
-import { AppBar, Button, Frame, Stack } from '@opencrvs/components'
-import { BackArrow } from '@opencrvs/components/lib/icons'
-import { Plus } from '@opencrvs/components/src/icons'
-import { ProfileMenu } from '@client/components/ProfileMenu'
+import { useLocation, useNavigate, matchPath } from 'react-router-dom'
+import {
+  useTypedParams,
+  useTypedSearchParams
+} from 'react-router-typesafe-routes/dom'
+import styled from 'styled-components'
+import { useSelector } from 'react-redux'
+import {
+  applyDraftToEventIndex,
+  deepDropNulls,
+  EventConfig,
+  EventIndex,
+  EventStatus
+} from '@opencrvs/commons/client'
+import {
+  APP_BAR_HEIGHT,
+  AppBar,
+  Button,
+  Frame,
+  Stack,
+  Icon
+} from '@opencrvs/components'
 import { useDrafts } from '@client/v2-events/features/drafts/useDrafts'
 import { useEventConfiguration } from '@client/v2-events/features/events/useEventConfiguration'
 import { useEvents } from '@client/v2-events/features/events/useEvents/useEvents'
 import { ActionMenu } from '@client/v2-events/features/workqueues/EventOverview/components/ActionMenu'
 import { useIntlFormatMessageWithFlattenedParams } from '@client/v2-events/messages/utils'
 import { ROUTES } from '@client/v2-events/routes'
-import { CoreWorkqueues, flattenEventIndex } from '@client/v2-events/utils'
-import { SearchToolbar } from '@client/v2-events/features/events/components/SearchToolbar'
+import { getLocations } from '@client/offline/selectors'
+import { flattenEventIndex } from '@client/v2-events/utils'
 import { DownloadButton } from '@client/v2-events/components/DownloadButton'
 import { recordAuditMessages } from '@client/i18n/messages/views/recordAudit'
-import { Sidebar } from '../sidebar/Sidebar'
-/**
- * Basic frame for the workqueues. Includes the left navigation and the app bar.
- */
+import { useUsers } from '@client/v2-events/hooks/useUsers'
+import { EventOverviewProvider } from '@client/v2-events/features/workqueues/EventOverview/EventOverviewContext'
+
+const Tab = styled.button`
+  border: none;
+  background: none;
+  color: ${({ theme }) => theme.colors.supportingCopy};
+  ${({ theme }) => theme.fonts.bold16};
+  cursor: pointer;
+  height: 100%;
+  box-sizing: border-box;
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.copy};
+  }
+
+  &.active {
+    color: ${({ theme }) => theme.colors.copy};
+    border-bottom: 3px solid ${({ theme }) => theme.colors.primary};
+    border-top: 3px solid transparent;
+  }
+`
```

Comment:
https://develop.opencrvs.pages.dev/?path=/docs/controls-tabs--docs
We do have this component, wonder what's it's use anymore / could we combine / remove one?

Reply:
The event overview layout also looks similar to what we have in de-duplication, with the comparison views and what not, if I remember correctly. Could we show an example of usage with `<Frame>` in the Storybook